src/content/guides/creating-your-own-admission-controller/index.md

```diff
@@ -0,0 +1,219 @@
++++
+title = "Creating your own admission controller"
+description = "Tutorial on how to deploy Istio on a Giant Swarm Kubernetes cluster."
+date = "2018-10-26"
+type = "page"
+weight = 50
+tags = ["tutorial"]
++++
+
+# Creating your own admission controller
```

Comment:
here and in line two please switch to title casing, I think we default to AP title case

Reply:
```suggestion
# Creating Your Own Admission Controller
```